packages/bvaughn-architecture-demo/components/sources/PointPanel.module.css

```diff
@@ -3,7 +3,7 @@
   flex-direction: column;
   background-color: var(--point-panel-background-color);
   padding: 1em;
-  gap: 1em;
+  gap: 0.4em;
```

Comment:
This is a matter of aligning to what the user sees when experiencing the app versus aligning to what the computer is computing. Sorry, in this case it's gotta be what the user sees. These are the numbers that lead to a clear and consistent visual design, whereas the 0.25 increments do not.

Reply:
As an outsider, it's not obvious how important `0.1em` is. 

Here are some questions that come to mind

1. what's the visual difference? side-by-side screenshots would help
2. is there a way to get `0.1em` elsewhere so that we preserve the `0.25em`
3. is there a good rubric for making this decision. Brian - it's not obvious what the impact of `0.25em` is. I've seen you discuss it elsewhere before, but forgot. Could we document it and if possible outline when the rule should be broken. Jon, why do you feel that 0.4em creates a more consistent visual design?

Reply:
> is there a good rubric for making this decision. Brian - it's not obvious what the impact of 0.25em is. I've seen you discuss it elsewhere before, but forgot. Could we document it and if possible outline when the rule should be broken. Jon, why do you feel that 0.4em creates a more consistent visual design?

I was trying to avoid [fractional pixels](https://www.simonbattersby.com/blog/browsers-and-fractional-pixels/), although that doesn't always happen because one of our font sizes is 10.

My main goal was to use it like a cheapy grid. _But_ because we're using `em` units (instead of `rem` units) it doesn't totally work that way in practice. (Side note: We could switch to using `rem` but then we'd have to confront the differences in root font size between my test harness and the legacy app – but maybe this would be worth it. If I recall, I _wasn't_ doing this because the legacy app used 15px and this doesn't divide nicely either.)

I think the biggest practical problems here are:
* Jon's probably noticing the inconsistencies between perceived padding that arise from the use of `em` and varying font sizes.
* I'm probably overly concerned about how difficult a lot of our old styles are to manage (lots of hard-coded color values, inconsistent sizing/units/fonts) and hoping to avoid a bunch of 1-off "tweaks" that will lead us back there.

Maybe worth talking about in a design meeting or something?